src/js/glightbox.js

```diff
@@ -1085,7 +1085,12 @@ class GlightboxInit {
                 let descHeight = description.offsetHeight;
                 let imgNode = image.querySelector('img');
 
-                imgNode.setAttribute('style', `max-height: calc(100vh - ${descHeight}px)`);
+                // if a slide height is set via data-height, we want to use that
+                // if not, we fall back to 100vh
+                const slideTriggerNode = this.elements[this.index].node;
+                const maxHeightValue = slideTriggerNode.getAttribute('data-height') ?? '100vh';
```

Comment:
The nullish coalescing operator (`??`) is not going to work in some of the supported browsers, which according to the README is 

> Safari
Mobile Safari
Opera
Edge
Firefox
Internet Explorer 11

Reply:
`??` seems to be supported in all modern browsers, but you're right, IE11 doesn't support it. 

https://caniuse.com/mdn-javascript_operators_nullish_coalescing

We could replace it with `||`, but we could also think about dropping IE11 as a whole, since Edge is the new default browser for Windows. No pressure though ✌🏻

If you prefer the `||` solution, I'll update my pull request tomorrow. Or do you have a completely different approach in mind?

Reply:
Also: isn't the code bundled and minfied using Rollup and Babel?
Because the string literals would also not work in IE11.

Babel should probably support compilation for legacy browsers as a default?

Reply:
Nope, you're right, it is covered by Babel's preset-env. It's been a while since I even looked in that folder :) @biati-digital  it all looks good to me

Reply:
Thanks for reviewing my code ✌🏻